lib/src/main/java/net/ypresto/androidtranscoder/MediaTranscoder.java

```diff
@@ -188,7 +200,11 @@ public void run() {
                         if (exception == null) {
                             listener.onTranscodeCompleted();
                         } else {
-                            listener.onTranscodeFailed(exception);
+                            if (exception instanceof InterruptedException) {
```

Comment:
Note that InterruptedException may be raised by cancel() or not (i.e. `Thread#interrupt()`).